src/content/docs/configuration/providers.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Providers
+description: Understand Providers and configure new Targets.
+sidebar:
+  label: Providers
+---
+
+Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
+
+1.  __Provider__  
+    A plugin that interfaces with [Daytona Server](/reference/server), responsible for Workspace deployment and lifecycle management.
```

Comment:
`with *the* Daytona Server`?

Reply:
Fixed in [1056c88](https://github.com/daytonaio/docs/pull/24/commits/1056c8807e7ee0cb48f25ebe8340a8036004b654)